spec/theme/plugins.spec.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+describe(\Dxw\MyTheme\Theme\Plugins::class, function () {
+    beforeEach(function () {
+        \WP_Mock::setUp();
+        \WP_Mock::wpFunction('esc_html', [
+            'return' => function ($a) {
+                return '_'.$a.'_';
```

Comment:
It's just so we can assert that our strings that contain user controlled data (plugin name in this case) have gone through our mocked implementation of `esc_html`

Do you think a better approach would be to mock each time we call `esc_html` with the arguments we expect it to receive?